R/oauth-init.R

```diff
@@ -109,10 +111,11 @@ init_oauth2.0 <- function(endpoint, app, scope = NULL, user_params = NULL,
 
   if (isTRUE(use_basic_auth)) {
     req <- POST(endpoint$access, encode = "form", body = req_params,
-      authenticate(app$key, app$secret, type = "basic"))
+      authenticate(app$key, app$secret, type = "basic"), config = config_init)
```

Comment:
Can you please convert all the calls you touched to use tidyverse style - http://style.tidyverse.org/syntax.html#long-lines